packages/oidc-middleware/package.json

```diff
@@ -32,7 +32,7 @@
     "express": "^4.16.3",
     "lodash": "^4.17.5",
     "negotiator": "^0.6.1",
-    "openid-client": "^2.1.0",
+    "openid-client": "2.1.0",
```

Comment:
Interesting - is this intentional because a newer version is not compatible?

Edit: Heh, unfortunately the comments you left don't line up anymore with the shas. But, did see your answer, 👍